test/test-uv-only.mjs

```
@@ -1,228 +1,244 @@
-
-/**
- * Test Python execution
- */
-async function testPython(pythonPath) {
-  console.log('Testing Python execution...');
+  console.log();
 
+  // -----------------------------------------------------------
+  // Test 2: Create venv with Python 3.13 (in temp dir)
+  // -----------------------------------------------------------
+  console.log('Test 2: Creating venv with Python 3.13...');
```

ESLint

[error] Unexpected console statement.

(no-console)